gruntfile.js

```diff
@@ -9,6 +9,14 @@ if (ab.enableTransform(module)) {
 module.exports = function(grunt) {
   require('load-grunt-tasks')(grunt);
 
+  grunt.registerTask('webdriverUpdateWithPinnedChromedriver', function() {
+    //ensures correct pinned ChromeDriver in config.js exists, otherwise fetch it.
+    if (!grunt.file.exists('./node_modules/webdriver-manager/selenium/chromedriver_' + config.chromedriverVersion)) {
+      console.log('Chromedriver ' + config.chromedriverVersion + ' not found. Running shell:webdriverUpdate.');
+      grunt.task.run('shell:webdriverUpdate');
+    }
+  });
```

Comment:
I'm somehwat ambivalent about the above.  Options.
1.  **Keep it the same as current develop** Do not do lines 12-18 and just always `shell:webdriverUpdate` with the chromeDriver version in config.js
2. **Leave the above change in**.  This will always check the **local chromedriver** against the chromeDriver version in config.js and update if needed using `webdriverUpdate`
3. **Enhance the above change** to check for **any** webdriver binary that is not matching a version in config.js and update if needed using `webdriverUpdate`.   [I have an experimental branch working here](https://github.com/blackboard/protractor-sync/compare/feature/experiment_from_update_latest_chrome?expand=1)
4.  **Revert pinning and just take the latest webdriver binaries** each time with no versions specified always running `webdriverUpdate`

Thoughts welcome!  I like #3 above myself but I'll go with what folks think.

Thanks!

Reply:
Option 3 looks good! Can you submit a new PR?

Reply:
thanks mindy!